### ISSUE TITLE
long echo/say lines

### ISSUE THREAD

**Issue by amstan** [amstan/hyperserv#19]
All echo and say commands that are long seem to get truncated for some reason, both ingame and irc.

A good test is say (useradmin).

**Comment by Calinou** [comment]
I have two ideas for this :
- When you do "#echo (bans)" (this is an example), send a link to the ban list. (it will avoid IRC excess'flood too)
- When you do the same command, add an argument to see 5 bans. "#echo (bans) 1" will echo the first 5 bans, 2 will send the next 5, and so.

**Comment by amstan** [comment]
echo will recieve a string.. how will it know where a ban starts and ends... echo does not work that way.
send a link to what? where is the banlist hosted on http?

**Comment by Calinou** [comment]
Yes... it's generated, and, after 15 minutes, it's deleted auto. It's sent by /notice, not public say. (unless you use #say)
